tests/sites/missing/devices/AHU-22/expected/errors.map

```diff
@@ -1,4 +1,3 @@
 Exceptions for AHU-22
-  While converting device config
-    While converting point filter_differential_pressure_sensor
-      point ref FAXLE12.present_value does not match expression ([A-Z]{2,4})([0-9]+)\.([a-z_]+)
+  While converting point filter_differential_pressure_sensor
```

Comment:
Do you know why it lost the "converting device config"... seems like it's losing a bit of important context... ?

Reply:
Restored this behavior. I had a new config generation function which omitted this text whereas the old did.